frontstage/views/surveys/add_survey_submit.py

```diff
@@ -49,4 +59,22 @@ def add_survey_submit(session):
     logger.info('Successfully retrieved data for confirm add organisation/survey page',
                 case_id=case_id, party_id=party_id)
     return redirect(url_for('surveys_bp.get_survey_list', _anchor=(business_party_id, added_survey_id), _external=True, business_party_id=business_party_id,
-                            survey_id=added_survey_id, tag='todo'))
+                            survey_id=added_survey_id, tag='todo', already_enrolled=already_enrolled))
+
+
+def is_business_enrolled(associations, survey_id, party_id):
+    """
+    Makes two checks as a business can be enrolled  on a survey but not the respondent, which would causes a single
```

Comment:
I don't think this docstring properly describes what is going on.  I think it should be something like this... but I don't know, if you could get 3rd person to make sure it makes sense then that would be helpful as this what this one function does is really important and the Why of this function is fairly important to explain to future people who come along to maintain this in the future.
`This function checks that the business AND the respondent are enrolled in a survey.  We first check if the respondent is enrolled, then if not, we check that the business is enrolled on the survey.  A business can have multiple respondents enrolled for a survey, and a respondent can be enrolled for multiple surveys.
If we only checked if the business was enrolled, then there could only ever be one respondent that could be able to answer the survey, which isn't the desired functionality.`

Reply:
```Ensures that
* Individuals are a associated with the business
* Business is enrolled in a survey```

?

Reply:
There's another thing to consider for this.  There are technically 2 'errors' that occur with this function.  A user could not be associated with the business, and the business could not be enrolled with the survey.  Currently, if either of those are false then we just go 'nah, you can't be enrolled on this survey'.  Maybe not now, but we should be more helpful with our error messages.